pkg/core/native/native_neo.go

```diff
@@ -167,6 +167,27 @@ func (n *NEO) increaseBalance(ic *interop.Context, h util.Uint160, si *state.Sto
 	if amount.Sign() == 0 {
 		return nil
 	}
+	oldAcc, err := ic.DAO.GetAccountState(h)
+	if err != nil {
+		return err
+	}
+	if err := n.ModifyAccountVotes(oldAcc, ic.DAO, (&big.Int{}).Not(&acc.Balance)); err != nil {
```

Comment:
It is not `Not`, but `Neg`.
We also use `new(big.Int)` in every other place.

Reply:
> It is not Not, but Neg.

Yep, it was fixed in some subsequent commit, maybe I need to shuffle these changes a little.